src/worker.ts

```diff
@@ -1,16 +1,7 @@
-/**
- * Welcome to Cloudflare Workers! This is your first worker.
- *
- * - Run `npm run dev` in your terminal to start a development server
- * - Open a browser tab at http://localhost:8787/ to see your worker in action
- * - Run `npm run deploy` to publish your worker
- *
- * Learn more at https://developers.cloudflare.com/workers/
- */
-
-import { env } from "./types/global.types";
+import { env } from './types/global.types';
 
 export default {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
 	async fetch(request: Request, env: env, ctx: ExecutionContext): Promise<Response> {
```

Comment:
please fix this why do we need `// eslint-disable-next-line @typescript-eslint/no-unused-vars`

Reply:
This variable is required for the variable to work and it is an auto-generated code.
Also we can not remove this code as it will help us check if our worker has been deployed properly